.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -76,3 +76,11 @@ body:
         ```
     validations:
       required: false
+  - type: textarea
```

Comment:
Partially duplicate of the 'Version' area. Would you care to rework this to only relate to additional packages? Discord.NET does not need to be specified.